packages/@aws-cdk/aws-lambda/lib/lambda.ts

```diff
@@ -226,4 +244,27 @@ export class Function extends FunctionRef {
             variables: this.environment
         };
     }
+
+    private buildDeadLetterConfigProperty(props: FunctionProps) {
```

Comment:
Sorry for being soooo annoying and picky, can you rename this to `buildDeadLetterConfig`? (concise names...)

Reply:
done